torch/distributed/_shard/sharded_tensor/utils.py

```diff
@@ -99,7 +98,21 @@ def build_metadata_from_local_shards(
     current_rank: int,
     pg: distributed_c10d.ProcessGroup
 ) -> ShardedTensorMetadata:
+    """
```

Comment:
Please remove the new dosctrings and only fix the errors listed in the issue. I think the issues clearly asks to skip the "Missing docstring" errors.